internal/e2e/sourcepkg/node.go

```diff
@@ -56,10 +59,15 @@ type Node struct {
 	M7 map[string]*Workload
 	M8 map[string]Workload
 
-	// S1 Workload  // for testing struct-to-struct for slices
-	// S2 *Workload // for testing ptr-to-ptr for slices
-	// S3 *Workload // for testing ptr-to-struct for slices
-	// S4 Workload  // for testing struct-to-ptr for slices
+	// T1 timestamp.Timestamp  // for testing struct-to-struct for time
+	// T2 *timestamp.Timestamp // for testing ptr-to-ptr for time
+	T3 *timestamp.Timestamp // for testing ptr-to-struct for time
```

Comment:
This is only handling pointer-protobufs to non-pointer-stdlib conversions for now, which is the most common.